modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/notifications/NotificationsInfo.js

```diff
@@ -119,19 +119,27 @@ const NotificationsInfo = ({
 	const [notificationName, setNotificationName] = useState(
 		selectedItem.data.notifications?.name?.[notificationIndex] || ''
 	);
+
 	const [notificationTypeEmail, setNotificationTypeEmail] = useState(
-		selectedItem.data.notifications?.notificationTypes?.[
+		selectedItem.data.notifications?.notificationTypes[notificationIndex][
```

Comment:
The crash described on the previous command was found by investigating how notificationTypeEmail would behave when creating new sections (and indexes, by consequence). `notificationTypes` is an array of arrays. The first array relates to section index, so it is ok to use `notificationIndex` variable, but the second array always have two elements/objects at max: one for email type, other for user notification type. Since the amount of sections is unlimited, this should break on the third section. You don't need to reference the index of the second array, just use `include` to check if there is an object inside the array with a value of "email" for the `notificationType` property.